cmd/egress/cmd.go

```diff
@@ -121,6 +129,68 @@ are set correctly before execution.
 				PlatformType: platformType,
 				Proxy:        p,
 			}
+			// Pod mode workflow
+			if config.podMode {
+				// Build a REST config using user-provided kubeconfig file
+				restConfig, err := clientcmd.BuildConfigFromFlags("", getKubeConfigPath(config.kubeConfigPath))
+				if err != nil {
+					fmt.Printf("could not build KubeAPI client due to error loading kubeconfig: %v\n", err)
+					os.Exit(1)
+				}
+
+				// Generate a KubeAPI clientset from the REST config
+				clientset, err := kubernetes.NewForConfig(restConfig)
+				if err != nil {
+					fmt.Printf("could not build KubeAPI client: %v\n", err)
+					os.Exit(1)
+				}
+
+				// Create a new KubeVerifier using the clientset
+				kubeVerifier, err := kubeverifier.NewKubeVerifier(clientset, config.debug)
+				if err != nil {
+					fmt.Printf("could not build kubeVerifier: %v\n", err)
+					os.Exit(1)
+				}
+
+				// HELLO WORLD EXAMPLE
+				fmt.Println("START POD MODE HELLO WORLD EXAMPLE")
+				kubeVerifier.KubeClient.SetNamespace("openshift-network-diagnostics")
+				kubeVerifier.KubeClient.CreateJob(context.TODO(), &batchv1.Job{
```

Comment:
So this is where we would inject a `podmode` probe that generates the job then hands it in yeah?